docs/api/en/objects/InstancedMesh.html

```diff
@@ -45,7 +45,10 @@ <h2>Properties</h2>
 		<h3>[property:Box3 boundingBox]</h3>
 		<p>
 			This bounding box encloses all instances of the [name]. Can be calculated
-			with [page:.computeBoundingBox](). Default is `null`.
+			with [page:.computeBoundingBox](). By default, this bounding box is used for
+			the purposes of frustum culling. You have to manually update the bounding box
```

Comment:
> By default, this bounding box is used for the purposes of frustum culling.

I would leave this sentence out. Bounding volumes are used for different purposes and it's not the task of the API documentation to explain or mentioned them.

Reply:
It's sufficient to highlight that bounding volumes should be recomputed whenever you transform instances. I would not like to see a list of use cases bounding volumes are intended for. They are versatile thing at it's not the task of a API documentation to educate computer graphics.

Reply:
When devs updating their geometry data without thinking about bounding volumes, they run into all sorts of issues. Failing view frustum culling is only one issue of many. Ray casting won't work as expected as well. Next collision detection tests. You have stumbled across one issue of series of issues but that does not mean view frustum culling is the most important thing of use case of bounding volumes.

We have the policy that guides in the manual are the best place for explaining such things.

Reply:
I understand your concerns. Let me think about where to document this topic. We have similar issue in other areas, see #27713.